### PR TITLE
Initial proposal draft for PyCon AU 2018

### PR DESCRIPTION
**Pull request by ncoghlan**

### CODE REVIEW

File: 2018/proposal.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+arrange to run a "Student Showcase", inspired by the well-received student
+presentations at PyCon Australia 2017.
+
+The general idea of this part of the proposal would be to:
+
+* Provide a way for teachers & senior secondary students to nominate as potential
+  presenters at the Showcase (similar to the way Gungahlin College students did
+  through the regular CfP for 2017)
+* Accept up to 6 projects for presentation (these would each be given a 15 minute
+  presentation slot *without* any question & answer time, and a ~5 minute break
```
Comment (nickyringland) [inline 2018/proposal.md]: I think it'd be good for students to get one question. Proposal: we have one question for each project prepared, and use it, or a question from twitter if there's a good one. I think having a bit of dialog with the students is useful, and answering questions is scary, so giving them a chance to do so in a supportive environment is well worth giving them only 12 or 13 minutes to present.

Comment (ncoghlan) [reply 2018/proposal.md]: Ah, moderated questions only - I like it.

Comment (nickyringland) [reply 2018/proposal.md]: That way we can reassure the kids that we'll only ask nice questions (i.e. things that they will already know) if they're worried about that sort of thing, and not like: "Why didn't you use <this framework you've never heard of>?" or similar.

File: 2018/proposal.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+For the final session, we'd like to work with the main conference organisers to
+arrange to run a "Student Showcase", inspired by the well-received student
+presentations at PyCon Australia 2017.
+
+The general idea of this part of the proposal would be to:
+
+* Provide a way for teachers & senior secondary students to nominate as potential
+  presenters at the Showcase (similar to the way Gungahlin College students did
+  through the regular CfP for 2017)
+* Accept up to 6 projects for presentation (these would each be given a 15 minute
```
Comment (ncoghlan) [inline 2018/proposal.md:53]: I just remembered that this math doesn't quite work, since the sessions are normally structured as 30+10+30+10+30=110 minutes, rather than 120 minutes (since we only need the 10 minute changeover either side of the middle talk, not before the first talk or after the third one).

Still, I think 5 or 6 shorter presentations will be a better option here than 3 regular length ones.

File: 2018/proposal.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  presentation slot *without* any question & answer time, and a ~5 minute break
+  in between slots. Moving back and forth between the Student Showcase and the other
+  specialist tracks would generally be discouraged)
+* Work with the conference organisers to provide a way for teachers, family members,
+  and students to attend just the showcase session for a relatively nominal cost
+  (e.g. $15)
```
Comment (nickyringland) [inline 2018/proposal.md]: This bit is a bit problematic I think. Teachers and students aren't going to like this. I don't think we'll be inundated with an influx of parents and friends coming along to watch the kids. It's a Thursday, after all.
From talking to schools, unfortunately $15 or $10 isn't necessarily a nominal cost. :( 

I'd be happier trying to get a sponsor to chip in for the cost of the kids' attendance for the showcase session if we need something there. I'm confident I can convince a company to chip in little bit to cover some extra afternoon tea for the kids, if that's what we need.

File: 2018/proposal.md (hunk 2 above)
Comment (nickyringland) [reply 2018/proposal.md:53]: Yup. I don't think we'll need 5 minutes change between each talk, either. We'll get the kids' slides in advance, put them all on one computer, deal with code demos as required... we can cut that down pretty easily. :)

File: 2018/proposal.md (hunk 3 above)
Comment (ncoghlan) [reply 2018/proposal.md]: Specialist tracks will be on the Friday.

I was thinking regardless of whether these were made generally available or not, accepted students should get a handful of zero-cost tickets to give out (e.g. themselves plus up to 5 others), so I'll reword this to make it clearer we have a lot of flexibility in choosing an approach we think will work well for teachers, students, families, friends, and the conference itself.

File: 2018/proposal.md (hunk 2 above)
Comment (ncoghlan) [reply 2018/proposal.md:53]: And use a handheld microphone to avoid the need to get lapel mikes sorted out.

File: 2018/proposal.md (hunk 3 above)
Comment (ncoghlan) [reply 2018/proposal.md]: I've switched the example out with one where each selected student would be given half a dozen Showcase-only tickets to distribute.

File: 2018/proposal.md (hunk 2 above)
Comment (ncoghlan) [reply 2018/proposal.md:53]: I made the math work by cutting the nominal transition time to 3 minutes. That gives a nominal 105 minutes total. Whatever we actually do in practice will need to be coordinated with the AV team anyway (since they need to know how to cut the videos appropriately), but I think this is fine for the proposal stage.

File: 2018/proposal.md (hunk 3 above)
Comment (nickyringland) [reply 2018/proposal.md]: True - sorry - Friday. Brain snap there. But still hard for students' friends/family to get time off school/work.

Yup, that sgtm. I don't even think they'll need that many, but I guess it doesn't hurt!

File: 2018/proposal.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+Note that while in 2017 there was some question as to whether PyCon Australia
+volunteer organisers needed to apply for and receive the appropriate state credentials
+for working with children due to some speakers and volunteers being under 18, adding
+the Student Showcase to the program would make this requirement unambiguous:
+since the Student Showcase would be intended specifically for high school students,
+the Education Seminar organisers and volunteers and the main conference organisers
+would definitely need those approvals in place. Organisers of other specialist tracks
+may not need them, but it would likely be simpler to make it a general requirement
+for all volunteers at the event.
```
Comment (nickyringland) [inline 2018/proposal.md:74]: The volunteer Working with Children Check (or other state equivalent) is free, too, which is nice.

Technically, we wouldn't need to [require WWCC](https://www.kidsguardian.nsw.gov.au/child-safe-organisations/working-with-children-check/about-the-check/exemptions) for those coming from out of state, but I would strongly recommend it anyway. 

> Interstate visitors:
> can work or volunteer at a one-off event such as a jamboree, sporting or religious event or tour, for up to 30 days a year without a NSW Working With Children Check
> can work or volunteer in any child-related work for up to 30 days a year, if the person holds an interstate Working With Children Check, or is exempt from the requirement to have such a check in his or her home jurisdiction

Comment (ncoghlan) [reply 2018/proposal.md:74]: The other states have similar requirements, so we still need the equivalent in our home state though. For example, https://www.bluecard.qld.gov.au/volunteers/Churchesclubsandassociationsinvolvingchildren.html seems to be the most applicable category here in Qld.

We determined in 2017 that we *didn't* need them because we weren't "conducting activities that mainly involve children". I believe that will change, at least for the Education Seminar, as soon as we introduce the Student Showcase. I don't think that's a problem - we just need to be aware of it far enough in advance that folks have plenty of time to apply for one, and to ensure we have the right information from Linux Australia to include on the form.
